modules/cas_cache/layer_cache_management.c

```diff
@@ -1327,7 +1327,6 @@ static int _cache_mngt_remove_core_flush(ocf_cache_t cache,
 {
 	int result = 0;
```

Comment:
This commit does more than just "refactoring" - more of a redesign. or maybe:
remove inactive core handling from "remove core"

Reply:
I rolled back most of the changes. Existing code was better.